sdks/go/pkg/beam/runners/prism/internal/environments.go

```diff
@@ -46,16 +47,26 @@ import (
 
 func runEnvironment(ctx context.Context, j *jobservices.Job, env string, wk *worker.W) error {
 	logger := j.Logger.With(slog.String("envID", wk.Env))
-	// TODO fix broken abstraction.
-	// We're starting a worker pool here, because that's the loopback environment.
```

Comment:
Was this TODO just obsolete? (I'm not seeing a pool started below, changed or not.)

Reply:
Yeah, it's mostly referring to the "one environment, one worker" notion. The "pool" for example could be several "w orkers" all in the same environment.

If we ever decide to make prism distributed and scalable, then this code would need to change anyway, so the TODO is moot.